lib/commons/dom/elements-from-point.js

```diff
@@ -40,15 +47,68 @@ dom.elementsFromPoint = function (doc, x, y) {
 
 		// add "pointer-events: none", to get to the underlying element
 		current.style.setProperty('pointer-events', 'none', 'important');
-
-		if (dom.isOpaque(current)) { break; }
 	}
 
 	// restore the previous pointer-events values
 	for (i = previousPointerEvents.length; !!(d = previousPointerEvents[--i]);) {
 		elements[i].style.setProperty('pointer-events', d.value ? d.value : '', d.priority);
 	}
 
+	elements = dom.reduceToElementsBelowFloating(elements, targetNode);
+
 	// return our results
-	return elements;
+	return dom.reduceToFirstOpaque(elements);
+};
+
+/**
+ * Reduce an array of elements to only those that are below a 'floating' element.
+ * 
+ * @param {Array} elements
+ * @param {Element} targetNode
+ * @returns {Array}
+ */
+dom.reduceToElementsBelowFloating = function (elements, targetNode) {
+	var floatingPositions = ['fixed', 'sticky'], finalElements = [], targetFound = false, index, currentNode, style;
+	
+	// Filter out elements that are temporarily floating above the target
+	for (index = 0; index < elements.length; ++index) {
+		currentNode = elements[index];
+		if (currentNode === targetNode) {
+			targetFound = true;
+		}
+
+		style = window.getComputedStyle(currentNode);
```

Comment:
Is it possible to get `style.position` directly instead of calling `window.getComputedStyle`? I want to make sure we aren't making performance worse than it already is for pages with lots of elements.